infrastructure-formula/infrastructure/salt/pistis.sls

```diff
@@ -0,0 +1,31 @@
+{%- from 'infrastructure/salt/map.jinja' import git -%}
+
+salt_pistis_package:
+  pkg.installed:
+    - name: pistis
+
+salt_pistis_files:
+  file.managed:
+    - names:
+        - /etc/systemd/system/salt-stage.path:
+            - source: salt://infrastructure/salt/files/etc/systemd/system/salt-stage.path
+            - mode: '0644'
+        - /etc/systemd/system/salt-stage.service:
+            - source: salt://infrastructure/salt/files/etc/systemd/system/salt-stage.service
+            - mode: '0644'
+        - /etc/pistis:
+            - contents:
+                - {{ pillar.get('managed_by_salt_formula', '# Managed by the infrastructure formula') | yaml_encode }}
+                - 'GITLAB_TOKEN={{ salt['pillar.get']('infrastructure:salt:pistis:gitlab_token', '') }}'
+            - mode: '0600'
+    - context:
+        branch: {{ git.branch }}
+        directory: {{ git.directory }}
+    - template: jinja
+
+salt_pistis_service:
+  service.running:
+    - name: salt-stage.path
+    - reload: False
+    - watch:
+        - file: salt_pistis_files
```

Comment:
Do we need a similar watch that does `systemctl daemon-reload` if `salt-stage.{path,service}` get changed?

Reply:
If the unit is active while it is being changed, then yes. However we already have centralized logic for that in https://github.com/openSUSE/heroes-salt/blob/production/salt/profile/systemd/daemon-reload.sls, hence I would rather not additionally implement it in pistis.sls.
I think moving daemon-reload.sls to a formula would be a good idea instead.

Reply:
Right, that makes a separate watch superfluous.

Having daemon-reload.sls as a separate formula sounds like something for another PR, therefore I'll resolve this here ;-)